src/main/kotlin/com/bridgecrew/ui/buttons/SeverityFilterButton.kt

```diff
@@ -33,17 +34,32 @@ class SeverityFilterButton(val project: Project,text: String, severity: Severity
                 g2d.dispose()
             }
         }
-        isEnabled = CheckovResultsListUtils.getCurrentResultsSeverities(project.service<ResultsCacheService>().checkovResults).contains(severity)
-    }
 
-    override fun updateUI() {
-        super.updateUI()
```

Comment:
how come we don't need this method? 😮

Reply:
We're basically creating the button every time 😅